ci/plugins/mzcompose/hooks/post-command

```diff
@@ -17,6 +17,9 @@ run() {
     bin/ci-builder run stable bin/mzcompose --mz-quiet --find "$BUILDKITE_PLUGIN_MZCOMPOSE_COMPOSITION" "$@"
 }
 
+# Run before potential "run down" in coverage
+xargs docker ps --all --quiet | docker inspect > docker-inspect.log
```

Comment:
xargs need to come after he pipe. This worked for me:

```
docker ps --all --quiet | xargs docker inspect
```

Reply:
Oops, of course.